src/graphic/Text.ts

```diff
@@ -741,6 +747,8 @@ class ZRText extends Displayable<TextProps> {
 
         if (textStroke) {
             subElStyle.lineWidth = retrieve3(tokenStyle.lineWidth, style.lineWidth, defaultLineWidth);
+            subElStyle.lineDash = retrieve3(tokenStyle.lineDash, style.lineDash, []);
+            subElStyle.lineDashOffset = style.lineDashOffset || 0;
```

Comment:
Perhaps the default `lineDash` should be `null`  instead of an empty array. Or in the `brush` method it will enter drawing `lineDash` branch https://github.com/ecomfe/zrender/pull/618/files#diff-0ac9120ba7869da49840beff4c68ab8eR315